flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/CalcITCase.scala

```diff
@@ -260,4 +260,17 @@ class CalcITCase extends StreamingTestBase {
     val expected = List("2,2,Hello", "3,2,Hello world")
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
   }
+
+  @Test
+  def testFromBase64(): Unit = {
```

Comment:
Can you add test in `ScalarFunctionsTest.testFromBase64` instead of it case?
Maybe you can add a nested function call to verify this.

Reply:
It's a good idea. Done.